r2dbc-spi-test/src/main/java/io/r2dbc/spi/test/MockRowMetadata.java

```diff
@@ -21,6 +21,9 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
```

Comment:
`stream.Collectors` isn't needed any more.

Reply:
Sorry I missed that.